README.rst

```diff
@@ -131,6 +131,12 @@ Transformer Engine comes preinstalled in the pyTorch container on
 From source
 ^^^^^^^^^^^
 
+For JAX, you need to install those dependencies first. (We are working to make this automatic):
```

Comment:
I don't think we need to say that we work to make this automatic - at some point we will just drop this section :-).